spec/controllers/idv/hybrid_mobile/socure/document_capture_controller_spec.rb

```diff
@@ -256,6 +256,40 @@
         expect(response).to redirect_to(idv_unavailable_path)
       end
     end
+    context 'reuse of valid capture app urls when appropriate' do
+      let(:fake_capture_app_url) { 'https://verify.socure.test/fake_capture_app' }
+      let(:socure_capture_app_url) { 'https://verify.socure.test/' }
+      let(:docv_transaction_token) { '176dnc45d-2e34-46f3-82217-6f540ae90673' }
+      let(:response_body) do
+        {
+          referenceId: '123ab45d-2e34-46f3-8d17-6f540ae90303',
+          data: {
+            eventId: 'zoYgIxEZUbXBoocYAnbb5DrT',
+            docvTransactionToken: docv_transaction_token,
+            qrCode: 'data:image/png;base64,iVBO......K5CYII=',
+            url: socure_capture_app_url,
+          },
+        }
+      end
+
+      before do
+        allow(request_class).to receive(:new).and_call_original
+        allow(I18n).to receive(:locale).and_return(expected_language)
+        allow(DocumentCaptureSession).to receive(:find_by).and_return(document_capture_session)
+      end
+
+      it 'does not create a DocumentRequest when valid capture app exists' do
+        document_capture_session.socure_docv_capture_app_url = fake_capture_app_url
+        document_capture_session.save
+        get(:show)
+        expect(request_class).not_to have_received(:new).
+          with(
+            redirect_url: idv_socure_document_capture_update_url,
+            language: expected_language,
+          )
```

Comment:
```suggestion
        expect(request_class).not_to have_received(:new)
```

i think the goal here is verify that new is not called at all. checking the params here, as is, it's possible this check could pass if new is called and the params were different